tool/grind.dart

```diff
@@ -208,7 +208,8 @@ updateTestPackageDocs() {
         '../test_package_docs',
         '--example-path-prefix',
         'examples',
-        '--auto-include-dependencies'
+        '--auto-include-dependencies',
+        '--pretty-index-json'
```

Comment:
IIRC this should produce a noticeable difference in the index.json in the code review, here.  Why doesn't it?